### PR TITLE
[FIX] barcodes: delay between keys

### PR DESCRIPTION
**Pull request by nim-odoo**
In order to detect if a barcode scanner is being used, we detect the
delay between keypresses. The value is hardcoded to 55 ms. However, in
the case of a bluetooth scanner, this delay is often not enough,
resulting in an incorrect capture of the barcode.

We add the config parameter `barcode.max_time_between_keys_in_ms` which
allows the user to fine tune this delay for his hardware.

opw-806398

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/barcodes/models/ir_http.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+from odoo import models
+
+
+class IrHttp(models.AbstractModel):
+    _inherit = 'ir.http'
+
+    def session_info(self):
+        res = super(IrHttp, self).session_info()
+        res['max_time_between_keys_in_ms'] = int(
+            self.env['ir.config_parameter'].sudo().get_param('barcode.max_time_between_keys_in_ms', default='55'))
```
Comment (sle-odoo) [inline addons/barcodes/models/ir_http.py]: don't you need to parseInt(x, 10)?

Comment (sle-odoo) [reply addons/barcodes/models/ir_http.py]: in the js i mean

Comment (nim-odoo) [reply addons/barcodes/models/ir_http.py]: not sure I understand your comment. there is a `int(` just above

Comment (sle-odoo) [reply addons/barcodes/models/ir_http.py]: didn't saw it, sorry about that :)

**Pull request merged**